cernopendata/modules/fixtures/data/records/cms-primary-datasets-Run2012B.json

```diff
@@ -223,7 +223,7 @@
       },
       {
         "description": "Getting started with CMS open data",
-        "url": "/getting-started/cms/2011"
+        "url": "/docs/cms-getting-started-2011"
```

Comment:
It seems better to me to upgrade the `/getting-started/...` redirection logic rather than the records metadata, ecause there may be many users out there using the old published URLs... Can you have a look at that?

Reply:
ok :)